x-pack/plugins/security_solution/public/common/components/page/manage_query.tsx

```diff
@@ -21,24 +19,21 @@ export interface OwnProps extends Pick<GlobalTimeArgs, 'deleteQuery' | 'setQuery
   inputId?: InputsModelId;
   inspect?: inputsModel.InspectQuery;
   legendPosition?: Position;
-  loading: boolean;
   refetch: inputsModel.Refetch;
-  session?: MutableRefObject<ISessionService>;
 }
 
 export function manageQuery<T>(
   WrappedComponent: React.ComponentClass<T> | React.ComponentType<T>
 ): React.FC<OwnProps & T> {
   const ManageQuery = (props: OwnProps & T) => {
-    const { deleteQuery, id, inspect = null, loading, refetch, setQuery, session } = props;
+    const { deleteQuery, id, inspect = null, refetch, setQuery } = props;
 
     useQueryInspector({
       deleteQuery,
       inspect,
-      loading,
+      loading: false,
```

Comment:
The `loading` flag won't be used anymore? 🤔 This HOC is widely used, is this change going to change the behavior everywhere?

Reply:
Yeah you're right, we should revert this change. This HOC is still in used by those components that fetch their own data.